Add 51Degrees to the list of IP location providers

jwrosewell opened the pull request:
Adds 51Degrees `High` and `Medium` confidence sources to the list of IP location providers. Note: Confidence is different to accuracy. 51Degrees can be confident a location result is within a large or small area.

jwrosewell committed:
Add 51Degrees to the list of location providers

Adds 51Degrees `High` and `Medium` confidence sources to the list of location providers. Note: Confidence is different to accuracy. 51Degrees can be confident a location result is within a large or small area.

jwrosewell committed:
Removed 51Degrees hyperlinks

Replaced link with core text to explain differences. Note: Low Confidence results should never be used with OpenRTB, AdCOM, or related processing. See https://51degrees.com/documentation/4.5/_ip_intelligence__features__confidence.html for more information.

jwrosewell commented:
@hillslatt could you let me know when this PR can be merged? Thanks.

hillslatt commented:
> @hillslatt could you let me know when this PR can be merged? Thanks.

As soon as we have a chance to connect and agree on what gets pushed. :)

jwrosewell committed:
Shortened 51Degrees definition

Modified the definition of 51Degrees to include the 51Degrees product name in brackets following the company name. Follows the format of Neustar, and reverses the format used by Digital Element.

jwrosewell commented:
Modified to shorten the description to just the product name in brackets.
